### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by robeeeert**
:kiss:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,12 +3,17 @@
   "version": "1.0.1",
   "description": "Node server which always responds with a 500 status.",
   "bin": "index.js",
-  "dependencies": {},
+  "dependencies": {
```
Comment (dan-lee) [inline package.json]: We should add this as devDependency :)

File: test/simple.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+const http = require('http')
+const { spawn } = require('child_process')
+
+describe('500 server', function () {
+  let server
+  let serverPromise
+
+  before(function () {
+    serverPromise = new Promise((resolve, reject) => {
+      server = spawn('node', ['index.js', '5000'])
```
Comment (dan-lee) [inline test/simple.js:11]: Great idea :)

**Review by dan-lee — CHANGES_REQUESTED** [review]

**Comment by dan-lee** [comment]
Great PR :+1:

File: package.json (hunk 1 above)
Comment (robeeeert) [reply package.json]: Of course, sorry :)

**Review by dan-lee — APPROVED** [review]

**Pull request merged**